shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/di/module/ViewModelModule.kt

```diff
@@ -1,9 +1,5 @@
 package co.nimblehq.avishek.phong.kmmic.di.module
 
-import co.nimblehq.avishek.phong.kmmic.presentation.module.*
-import org.koin.core.module.dsl.factoryOf
-import org.koin.dsl.module
+import org.koin.core.module.Module
 
-val viewModelModule = module {
-    factoryOf(::SplashViewModel)
-}
+expect val viewModelModule: Module
```

Comment:
Using the `androidx.lifecycle.ViewModel` requires us to set up ViewModel dependencies in both iOSMain and AndroidMain, is that correct?

Reply:
@phongvhd93 that's correct. `BaseViewModel` on Android needs to extend that class you mentioned for the VM to work properly on Android.